src/components/MessageList/MessageList.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import MessageList from './MessageList';
+
+jest.mock('../../hooks/useVideoContext/useVideoContext', () => () => ({
+  room: { localParticipant: { identity: 'olivia' } },
+}));
+
+const arbitraryDate = new Date();
```

Comment:
I think a hardcoded date could fix the failing Ci build (I think it must be time zone related). 

Something like: `new Date(1614634697422)`